File: atividade_nicolas/alunos/nicolas_g2/vitor_araujo/piscina.js
import prompt from 'prompt-sync'
const input = prompt()


function main(){

    const [largura, altura, profundidade] = input('Digite a largura, altura e profundidade da piscina, separados por espaços e em centímetros: ').split(' ').map(Number)

    const capacidade_maxima_piscina = (largura*altura*profundidade)/1000

    const capacidade_recomendada = capacidade_maxima_piscina * 0.85

    console.log(`\tÉ recomendado que a piscina seja enchida até ${capacidade_recomendada} litros`)

    const preco_1000litros = Number(input('Digite o valor a ser pago por cada 1000 litros: '))

    
    // Calculo do preço de acordo com a quantidade de litros

    let litrosPagar = capacidade_recomendada/1000

    let excedente = litrosPagar - Math.trunc(litrosPagar)
 
    let inteirosPagar = Math.trunc(litrosPagar)

    if(excedente > 0){
        inteirosPagar++
    } 

    let valor_a_ser_pago = inteirosPagar * preco_1000litros

    console.log(`\tVocê gastará R$ ${valor_a_ser_pago} para encher sua piscina até a capacidade recomendada`)

    const evaporacao = capacidade_recomendada * 0.1 //valor em litros

    // Calculo do preço de acordo com a quantidade de litros
    litrosPagar = evaporacao/1000

    excedente = litrosPagar - Math.trunc(litrosPagar)

    inteirosPagar = Math.trunc(litrosPagar)

    if (excedente > 0){
        inteirosPagar++
    }

    valor_a_ser_pago = inteirosPagar * preco_1000litros

    console.log(`\tMensalmente você gastará R$ ${valor_a_ser_pago} para manter a piscina no nível ideal`)

} 




main()

File: atividade_nicolas/alunos/nicolas_g2/giovana_lima/piscina.py
def main():
    print('\t Indicações para piscina -->')
    largura = float(
        input('Escreva a medida da largura da piscina em cm: '))
    comprimento = float(
        input('Escreva a medida da comprimento da piscina em cm: '))
    profundidade = float(
        input('Escreva a medida da profundidade da piscina em cm: '))

    capacidade = (largura * comprimento * profundidade)/1000
    capacidade_recomendada = 0.85 * capacidade
    agua_evaporada = 0.10 * capacidade_recomendada

    print(f'\nRecomenda-se encher a piscina até {capacidade_recomendada:.2f} litros.\n')

    valor_litro = float(input('Escreva o valor de cada 1000 litros de água: R$ '))     
    sim = capacidade_recomendada % 1000
    valor_a_ser_pago = 0

    if sim == 0:
        valor_a_ser_pago = (capacidade_recomendada // 1000) * valor_litro
    else:
        valor_a_ser_pago = ((capacidade_recomendada // 1000) + 1) * valor_litro

    valor_manutenção = (agua_evaporada//1000) * valor_litro

    print(f'\nValor a ser gasto para encher a piscina: R$ {valor_a_ser_pago:.2f}.')
    print(f'Valor mensal para manter piscina no nível ideal: R$ {valor_manutenção:.2f}')

if __name__ == '__main__':
    main()
